docs/.vuepress/theme/components/Sidebar.vue

```diff
@@ -64,6 +64,24 @@ export default {
   },
   mounted() {
     this.scrollToActiveElement(this.$route.path);
+    // THIS IS hack but it works
```

Comment:
Do we have a plan to replace it with something more elegant in the future?

Reply:
I spent 4 hours into trying to achive with Vue 1 sidebar component and failed

imho There is no way to extends this component https://github.com/vuejs/vuepress/blob/v1.9.10/packages/%40vuepress/theme-default/components/SidebarLink.vue and you would need to create your own. This requires an advanced vue delevoper